crates/e2e/src/setup/mod.rs

```diff
@@ -74,7 +74,7 @@ fn with_default_filters<T>(custom_filters: impl IntoIterator<Item = T>) -> Vec<S
 where
     T: AsRef<str>,
 {
-    let mut default_filters: Vec<_> = DEFAULT_FILTERS.into_iter().map(String::from).collect();
+    let mut default_filters: Vec<_> = DEFAULT_FILTERS.iter().map(|s| s.to_string()).collect();
```

Comment:
why this change? :thinking:

Reply:
Whenever you want to add or remove a log filter from that list you also have to update its type because it's a statically sized array. With this change I convert the thing into a fat pointer which knows its own length so you don't have to jump through these hoops anymore. Technically not needed for this PR but it annoyed me yet again so I just wanted to finally fix it.

Reply:
oh, I understand your point, got it! :ok_hand: thanks for the explanation!